cnc_ctrl_v1/Axis.h

```diff
@@ -50,7 +53,10 @@
             double     pidInput();
             double     pidOutput();
             long  steps();
-            
+			char  moved, stalled;
+			long  stalldelta;
```

Comment:
Is this ever used?  I only see it being set a value, never being used as a value.